FordLabs/retroquest
# Delete confirmation

## Overview
Removes the rude window confirmation dialog when deleting an action item and instead, shows an inline deletion overlay.

### Demo
<img width="351" alt="screen shot 2018-09-14 at 2 14 41 pm" src="https://user-images.githubusercontent.com/6293337/45567661-6d7e5a80-b828-11e8-8173-c8c8110109d0.png">
<img width="860" alt="screen shot 2018-09-14 at 2 14 53 pm" src="https://user-images.githubusercontent.com/6293337/45567669-7111e180-b828-11e8-8474-b9eb6d082f91.png">

## Testing Instructions
1. Add a thought and action item.
2. Click on the trash icon.
3. See the new overlay.
4. Click Yes, No, or off of the item, it should behave as you expect.
